docker-compose.yml

```
@@ -24,10 +24,17 @@ services:
       - FEATURE_PHASE_BANNER=False
       - WAGTAIL_API_URL=https://wagtail.nationalarchives.gov.uk/api/v2
       # - WAGTAIL_API_URL=http://host.docker.internal:8000/api/v2
+      - DELIVERY_OPTIONS_API_URL=http://mock-delivery-options-api:8080/rosetta/data
     ports:
       - 65533:8080
     volumes:
       - ./:/app
+  mock-delivery-options-api:
+    image: wiremock/wiremock:3.13.1
+    volumes:
+      - ./wiremock:/home/wiremock
+    ports:
+      - 8080:8080
```

I know I originally just mapped port `8080` but could we please change this to `65501` (so it doesn't conflict with other applications as `8080` is a popular port)?

This won't affect your `DELIVERY_OPTIONS_API_URL` because `mock-delivery-options-api` maps to the container and that will have Wiremock running on `8080` but just means if you want to see the API responses in your browser, you need to use http://localhost:65501.

I've already added the port to the list in https://national-archives.atlassian.net/wiki/spaces/TW/pages/775028742/Local+development

```suggestion
      - 65501:8080
```